sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -131,8 +131,6 @@ gnrc_rpl_instance_t *gnrc_rpl_instance_get(uint8_t instance_id)
 
 bool gnrc_rpl_dodag_init(gnrc_rpl_instance_t *instance, ipv6_addr_t *dodag_id, kernel_pid_t iface)
 {
-    /* TODO: check if netif_addr belongs to iface */
```

Comment:
For a new DODAG this doesn't work (I ran into an assert on a DIO receive there), so I moved [it outside](https://github.com/RIOT-OS/RIOT/pull/8128/files#diff-7688fe4f626e93c827c7d5b563b80aa5R574)

Reply:
Ack